pkg/ui/src/views/statements/statementsPage.tsx

```diff
@@ -181,12 +181,15 @@ export const selectStatements = createSelector(
     let statements = flattenStatementStats(state.data.statements);
     if (props.params[appAttr]) {
       let criteria = props.params[appAttr];
+	  let showInternal = false;
```

Comment:
nit: looks like the indentation got shifted?

Reply:
Done.